.github/actions/setup-terraform/action.yaml

```diff
@@ -0,0 +1,87 @@
+name: Setup Terraform
```

Comment:
Note that this action has a lot of logic in common with [the equivalent action](https://github.com/ccao-data/model-sales-val/blob/main/.github/actions/setup-terraform/action.yaml) in `model-sales-val`. When I copy the workflows from this repo over to the condo model in https://github.com/ccao-data/model-condo-avm/issues/5 I may decide to factor this logic out into an action that can be reused by workflows across repos; in the meantime, however, we accept a little bit of cross-repo duplication in order to iterate more quickly.

Reply:
I think the pattern in [r-lib/actions](https://github.com/r-lib/actions) works pretty well. Minor versions can change the code underlying the tag and only breaking changes would warrant a major version bump. I think it makes sense to basically factor out stuff into composite actions in their own sub-directories. Curious what you think

Reply:
I'm down to try it! Conceptually it seems kind of strange to me that a new major version would be applied to all packages in the repo even if breaking changes were only introduced in a subset of those packages, but if it's a common pattern in practice then I think it makes sense for us to follow it.